test/test_functional_tensor.py

```diff
@@ -141,7 +141,13 @@ def test_rotate_batch(self, device, dt):
     def test_rotate_deprecation_resample(self):
         tensor, _ = _create_data(26, 26)
         # assert deprecation warning and non-BC
-        with pytest.warns(UserWarning, match=r"Argument resample is deprecated and will be removed"):
+        with pytest.warns(
+            UserWarning,
+            match=re.escape(
+                "Parameter 'resample' is deprecated since 0.12.0 and will be removed 0.14.0. "
```

Comment:
also tests are failing
```suggestion
                "The parameter 'resample' is deprecated since 0.12.0 and will be removed 0.14.0. "
```